### PR TITLE
Enable dotnet-format

### LINKED ISSUE #237: Apply uniform style guidelines to repository
... while it's still small. Let's add it to the interop project while we're at it.

### PR DESCRIPTION
**Pull request by eddynaka**
Fixes #237

### CODE REVIEW

File: .github/workflows/dotnet-format.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  check-format:
+    runs-on: windows-latest
+
+    steps:
+    - name: check out code
```
Comment (lgolding) [inline .github/workflows/dotnet-format.yml:12]: >c [](start = 12, length = 1)

check => Check

File: .github/workflows/dotnet-format.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      uses: actions/checkout@v2
+
+    - name: Setup .NET Core 3.1
```
Comment (lgolding) [inline .github/workflows/dotnet-format.yml:15]: >Setup [](start = 12, length = 5)

"Setup" => "Set up". ("Setup" is a noun. "Set up" is a verb.)

File: .github/workflows/dotnet-format.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.x
+
+    - name: Install format tool
+      run: dotnet tool install -g dotnet-format
+
+    - name: dotnet format
```
Comment (lgolding) [inline .github/workflows/dotnet-format.yml:23]: >dotnet format [](start = 12, length = 13)

I suggest friendly names for all the steps, so if the previous step is "Install format tool", then this step should be "Run format tool".

File: src/.editorconfig [hunk 4]
```diff
@@ -0,0 +1,207 @@
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
```
Comment (lgolding) [inline src/.editorconfig:1]: ># [](start = 0, length = 1)

I you tell me you copied this from the SARIF SDK then I won't review it line by line. I reviewed the SARIF SDK's .editorconfig file when it was first added.

**Review by michaelcfanning — APPROVED** [review]
:shipit:

File: src/Sarif.Viewer.VisualStudio/ContentTypes.cs [hunk 5]
```diff
@@ -3,9 +3,10 @@
 
 namespace Microsoft.Sarif.Viewer
 {
-    using Microsoft.VisualStudio.Utilities;
     using System.ComponentModel.Composition;
```
Comment (michaelcfanning) [inline src/Sarif.Viewer.VisualStudio/ContentTypes.cs:6]: >System [](start = 10, length = 6)

Eddy, I went ahead and merged this. Thanks for getting this in. @v-lgold@microsoft.com, if you want to tinker further with this, go ahead and make the changes you'd like.

**Pull request merged**